models/onepunchswap/bnb/onepunchswap_bnb_normal_trades.sql

```diff
@@ -0,0 +1,111 @@
+{{ config(
+    schema = 'onepunchswap_bnb',
+    alias = 'normal_trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook='{{ expose_spells(\'["bnb"]\',
+                                "project",
+                                "onepunchswap",
+                                \'["hosuke"]\') }}'
```

Comment:
Hi @sunopar, thank you for submitting this PR for onepunchswap.
We may need some time to review the PR in details.
Since you are the author for these spells, you can rewrite the contributor's name, i.e:
```suggestion
                                \'["sunopar"]\') }}'
```

Also in the [schema.yml file](https://github.com/duneanalytics/spellbook/pull/3150/files#diff-645eb08d5d0cf8dd05968b1b9026af88ff6d68864cedd9c586cde3bf8cce2370R9): .